### PR TITLE
GameDB: Rayman Revolution fixes + missing demo entry

### PR DESCRIPTION
**Pull request by atomic83GitHub**
This PR add an Emotion Engine Rounding fix Rayman Revolution (nearest).
- Tested by @atomic83github @prafullpcsx2 and @Helium-4

It also add a missing demo entry along with dedicaced comments for some demo issues.

Close: https://github.com/PCSX2/pcsx2/issues/2452

### CODE REVIEW

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -126,6 +126,16 @@ Name   = Gran Turismo 3 - A-Spec [PS2 Bundle]
...
 MemCardFilter = PBPX-95503/SCUS-97102
 ---------------------------------------------
+Serial = PBPX-95506
+Name   = Playstation 2 - Demo Disc 2001
+Region = PAL-M5
+[patches = 7ACF7E03]
+comment=- You must enable the EETimingHack when playing the WRC demo to avoids random hangs.
+comment=- You must change the EE Clamping mode to Full when playing the Klonoa 2 demo to avoids misplaced objects.
+comment=- You must change the VU Clamping mode to extra when playing the Klonoa 2 demo to fixes reflections issues.
+comment=- You must use the ZeroSPU2 plugin when playing the Klonoa 2 demo to avoids bad/missing sounds.
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Tab whitespaces on the comment lines please.

**Comment by lightningterror** [comment]
Also add **GameDB** in the commit title.

File: bin/GameIndex.dbf [hunk 2]
```diff
@@ -126,6 +126,16 @@ Name   = Gran Turismo 3 - A-Spec [PS2 Bundle]
...
 MemCardFilter = PBPX-95503/SCUS-97102
 ---------------------------------------------
+Serial = PBPX-95506
+Name   = Playstation 2 - Demo Disc 2001
+Region = PAL-M5
+[patches = 7ACF7E03]
+comment=-You must enable the EETimingHack when playing the WRC demo to avoids random hangs.
+comment=-You must change the EE Clamping mode to Full when playing the Klonoa 2 demo to avoids misplaced objects.
+comment=-You must change the VU Clamping mode to extra when playing the Klonoa 2 demo to fixes reflections issues.
+comment=-You must use the ZeroSPU2 plugin when playing the Klonoa 2 demo to avoids bad/missing sounds.
```
Comment (lightningterror) [inline bin/GameIndex.dbf]:
```
[patches = 7ACF7E03]
	comment=
	comment=
	comment=
	comment=
[/patches]
```

**Comment by lightningterror** [comment]
It's still the same.

**Comment by atomic83GitHub** [comment]
Ready to merge :).

**Comment by lightningterror** [comment]
I still don't see whitespace added on the lines with the commends, it should look something like this.

> [patches ]
> 	comment=
> 	comment=
> 	comment=
> 	comment=
> [/patches]

**Comment by lightningterror** [comment]
Github didn't show the comment properly, anyway this is what I mean.

image

**Comment by atomic83GitHub** [comment]
Done.

**Pull request merged**